package.json

```diff
@@ -26,7 +26,7 @@
     "@babel/preset-env": "7.9.0",
     "@babel/register": "7.9.0",
     "@jest/test-sequencer": "^25.0.0",
-    "@wordpress/e2e-test-utils": "4.3.1",
+    "@woocommerce/e2e-utils": "file:tests/e2e-utils",
```

Comment:
@tammullen This will need to be updated to `tests/e2e/utils`.

Reply:
In addition to package.json, the package should be added to https://github.com/woocommerce/woocommerce/blob/master/webpack.config.js .

Reply:
@rrennick As discussed, we can address this in a separate PR